src/app/documentation/demos/nav/layout-subnav-primary.html

```diff
@@ -73,6 +73,14 @@
         </div>
     </div>
 </div>
+<clr-alert>
+    <div class="alert-item">
+        <span class="alert-text">
+            For code examples on the specific navigation components (header, subnav, sidenav)
+            see <a routerLink="/documentation/header">header</a> documentation.
+        </span>
+    </div>
+</clr-alert>
```

Comment:
Seems like the code duplication could have been mitigated by making this its own component?

Reply:
Yeah. I could make it a simple component with just the alert in it. Should I?

Reply:
It's probably easier to do it now...

Reply:
Done!